### PR TITLE
Add ApplyWowResampling

### PR DESCRIPTION
**Pull request by atamazian**
Implementing https://github.com/iver56/audiomentations/issues/57.

### CODE REVIEW

**Comment by iver56** [comment]
I'm on vacation, so my time is limited these days. I'll get to this eventually. Meanwhile, thanks for your patience :)

**Comment by iver56** [comment]
The implementation here seems to do something else than what is intended. Wow and flutter is a change of pitch in the audio over time. For an example, listen to [this youtube video](https://www.youtube.com/watch?v=kCwRdrFtJuE) from 0:37. The implementation in this pull request mixes in a distorted variant of the original signal, which is not the same thing as wow and flutter.

I can see how you were confused by the referenced paper, which states

bilde

However, the `x` mentioned here refers to a position (i.e. time) in the audio file, not the array of audio samples.

In other words, that function maps a position in the original audio to a position in the output audio.

To be able to sample smoothly from different positions in the audio, you would have to do some interpolation (e.g. sinc) to make it sound good.

**Comment by atamazian** [comment]
I've got the idea.

**Comment by iver56** [comment]
Do you plan to update this pull request? Or should I close it for now?

**Comment by atamazian** [comment]
Close it for now. Maybe later I'll finish this.
